spec/features/arclight_spec.rb

```diff
@@ -84,6 +84,11 @@
           expect(page).to have_css('h3 a', text: 'Place')
           expect(page).to have_css('li .facet-label', text: 'Mindanao Island (Philippines)', visible: false)
         end
+
+        within('.blacklight-access_subjects_sim') do
+          expect(page).to have_css('h3 a', text: 'Subject')
+          expect(page).to have_css('li .facet-label', text: 'Fraternizing', visible: false)
```

Comment:
Sorry that I wasn't clear. I'm saying to add the other access_subjects values to the spec so that we can verify the indexing code is picking them out of all those elements, e.g.,

```ruby
expect(page).to have_css('li .facet-label', text: 'Societies', visible: false) # from <subject/>
expect(page).to have_css('li .facet-label', text: 'Fraternizing', visible: false) # from <function/>
expect(page).to have_css('li .facet-label', text: 'Photographs', visible: false) # from <genreform/>
expect(page).to have_css('li .facet-label', text: 'Medicine', visible: false) # from <occupation/>
```